### PR TITLE
PICARD-1546: Truncate tracks in Other Releases menu when too many media

### PR DESCRIPTION
**Pull request by zas**
# Summary

* This is a…
    * [x] Bug fix
    * [ ] Feature addition
    * [ ] Refactoring
    * [ ] Minor / simple change (like a typo)
    * [ ] Other
* **Describe this change in 1-2 sentences**:

# Problem

When releases have a lot of media the list of number of tracks can be far too long for clean display

* JIRA ticket (_optional_): [PICARD-1546](https://tickets.metabrainz.org/browse/PICARD-1546)

# Solution

Truncate it if > 10

Capture du 2019-07-01 13-44-05

# Action

### CODE REVIEW

File: picard/releasegroup.py [hunk 1]
```diff
@@ -33,6 +33,9 @@
 from picard.util import uniqify
 
 
+MAX_TRACKS = 10
```
Comment (Sophist-UK) [inline picard/releasegroup.py]: MAX_MEDIA_DISCS? Not tracks anyway.

Should this be in picard/picard/const/__init__.py ?

File: picard/releasegroup.py [hunk 2]
```diff
@@ -74,6 +77,10 @@ def _parse_versions(self, document):
 
             countries = countries_from_node(node)
 
+            if len(node['media']) > MAX_TRACKS:
+                tracks = "+".join([str(m['track-count']) for m in node['media'][:MAX_TRACKS]]) + '+…'
```
Comment (Sophist-UK) [inline picard/releasegroup.py]: How about:
```
tracks = "+".join([str(m['track-count']) for m in node['media'][:MAX_TRACKS]]) + '+… (' + _('Total discs') + len(node['media']) + ')'
```

File: picard/releasegroup.py (hunk 1 above)
Comment (zas) [reply picard/releasegroup.py]: It's displayed in "Tracks" field hence the name.
And no, it should stay here as it's very specific to the code in this file. I'd perhaps even make it more local.

File: picard/releasegroup.py (hunk 2 above)
Comment (zas) [reply picard/releasegroup.py]: Unneeded. Media are already listed in "Format" field (ie. 100xCD)

Comment (Sophist-UK) [reply picard/releasegroup.py]: Ok.

File: picard/releasegroup.py (hunk 1 above)
Comment (Sophist-UK) [reply picard/releasegroup.py]: Local = OK. You know best.

**Comment by bdb484** [comment]
I've been advised to report a bug:
https://stackoverflow.com/a/56740829/3833085

I assume this is not the normal channel for reporting bugs, but I can't
figure out how else to do it.

**Comment by Sophist-UK** [comment]
@bdb484 - Please report bugs using the tracker at https://tickets.metabrainz.org

**Pull request merged**